alchemist-full/build.gradle.kts

```diff
@@ -145,6 +146,43 @@ tasks.register<Exec>("testJpackageOutput") {
     finalizedBy(deleteJpackageOutput)
 }
 
+val generatePKGBUILD by tasks.registering(Exec::class) {
+    group = "Publishing"
+    description = "Generates a valid PKGBUILD by replacing values in the template file"
+    val inputFile = file("${project.projectDir}/PKGBUILD.template")
+    val outputDir = file(rootProject.layout.buildDirectory.map { it.dir("package") }.get().asFile.path)
+    val tokenToReplace = "{%}"
+    var replacementValues: List<String>
+    val testSourceParam = System.getProperty("generatePKGBUILD.testSource", "false")
+    val interceptOutput = ByteArrayOutputStream()
+    standardOutput = interceptOutput
+    doFirst {
+        require(!isWindows && !isMac)
```

Comment:
Why? Generating a pkgbuild does not seem to be something requiring a specific OS